src/views/features/sole-trader/where-do-you-live/where-do-you-live.njk

```diff
@@ -52,14 +52,28 @@
     }
   ]
 }) }}
-    {{ govukButton({
-      text: i18n.SaveAndContinue,
-      id:"save-continue-button"
-    }) }}
+
+{% if reqType === "updateAcsp" %}
+          <div class="govuk-button-group">
+             {{ govukButton({
+                text: i18n.Continue,
+                id:"continue-button-id"
+             })}}
+             <a class="govuk-link id="previous-page-id" href={{previousPage}}>{{i18n.Cancel}}</a>
```

Comment:
Looks like you are missing a closing " on the class of this anchor tag.
Would need to update it like so:

<a class="govuk-link**"** id="previous-page-id" href={{previousPage}}>{{i18n.Cancel}}</a>